source/AndroidXProject.cshtml

```diff
@@ -71,6 +71,7 @@
   <ItemGroup>
     <None Include="@(Model.NuGetPackageId).targets" Pack="True" PackagePath="build\$(TargetFramework)" />
     <None Include="@(Model.NuGetPackageId).targets" Pack="True" PackagePath="buildTransitive\$(TargetFramework)" />
+    <None Include="..\..\LICENSE" Pack="True" PackagePath="LICENSE" />

```

Comment:
The build is failing with:

```
The license file 'LICENSE' does not exist in the package
```

It could be that the relative path is wrong, or the fact that there is no extension so it is actually placing the file at `LICENSE\LICENSE` instead.